alpine-nodejs/Dockerfile

```diff
@@ -2,18 +2,19 @@ FROM unocha/alpine-base-nodejs:latest
 
 MAINTAINER Serban Teodorescu <teodorescu.serban@gmail.com>
 
-ENV NODE_APP_DIR=/srv/www
+ENV NODE_APP_DIR=/srv/www \
+    PORT=3000
 
-COPY run_node package.json server.js /
+COPY run_node package.json server.js /tmp/
```

Comment:
Copied to /tmp instead or / to overcome a bug in certain docker version when building.